src/libDirectoryService/ShardingPostProcessing.cpp

```diff
@@ -251,11 +251,32 @@ bool DirectoryService::ProcessShardingConsensus(
     // So, ANNOUNCE should acquire a lock here
 
     lock_guard<mutex> g(m_mutexConsensus);
-
+#if 1 //clark
```

Comment:
Maybe we can remove if 1 and remove older codes

Reply:
Fixed in a549b5576bea69b5c54149ac68a26af5474e9123